src/slices/AccountSlice.ts

```diff
@@ -42,23 +42,27 @@ export const getBalances = createAsyncThunk(
   async ({ address, networkID, provider }: IBaseAddressAsyncThunk) => {
     const ohmContract = new ethers.Contract(addresses[networkID].OHM_ADDRESS as string, ierc20Abi, provider) as IERC20;
     const ohmBalance = await ohmContract.balanceOf(address);
+    console.log(ohmBalance);
```

Comment:
You might want to remove these console logs

Reply:
good call